tests/test_notify_slack.py

```diff
@@ -113,6 +131,54 @@ def should_not_notify_slack_when_step_function_running(webhook_client_mock: Magi
         webhook_client_mock.assert_not_called()
 
 
+@patch("backend.notify_status_update.task.WebhookClient.send")
+@patch("backend.notify_status_update.task.get_import_status_given_arn")
+def should_not_notify_slack_when_step_function_failed(
+    step_func_status_mock: MagicMock, webhook_client_mock: MagicMock
+) -> None:
+    # Given
+
+    webhook_client_mock.return_value.status_code = HTTPStatus.OK
+    mock_slack_url = any_https_url()
+    now_ts = datetime.now()
```

Comment:
Using the actual current datetime in tests is often a problem, in particular because it means the test is guaranteed not reproducible. Would it be possible to use a static datetime here, such as `datetime(2001, 2, 3, hour=4, minute=5, second=6, microsecond=789876, tzinfo=timezone.utc)` used elsewhere? Same in `should_notify_slack_with_finished_details_when_url_set`.